python/parquet_tools/__init__.py

```diff
@@ -0,0 +1 @@
+# This is a self-contained package with only one entry via the CLI.
```

Comment:
OK, I'll have to research the right way sub-packages are supposed to be configured here. Suggestions are welcomed.

Reply:
Just make a folder `python/parquet_tools/csv2pq` and move your modules there (and rename them).

Reply:
Sure, but what if it's counterpart pq2csv wanted to reuse one of the modules? It's sort of a naming conundrum. Would you just put pq2csv in with csv2pq? Also, what about the file in bin.src? Does it go into the same directory or stays outside?

Reply:
If you are not using scons but using setup.py you should not be using bin.src.  You should be using the standard setup.py scheme of specifying executable scripts (and they can go in a scripts directory).

Reply:
Naming/packaging is one of the two hard problems in CS :) I'd say that `pq2csv` should be either a separate sub-package or a module. If there is anything common between `csv2pq` and `pq2csv` then it can go in a `common` module/sub-package. It really is up to you, I just made an observation about existing module names.